nicoleWojewoda/assignments/treesBFS/lc637/lc637.js

```diff
@@ -0,0 +1,82 @@
+// Problem Statement #
+
+// Given a binary tree, populate an array to represent the averages of all of its levels.
+
+/*
+input: the root node of a binary tree
```

Comment:
Nice overview here. Would also suggest writing some additional test cases and also analyzing the time and space complexity of the problem.